Error: Unable to access jarfile /Users/x/Desktop/kotlin/backend/0_scripts/server/build/libs/server-all.jar
Error: Unable to access jarfile /Users/x/Desktop/kotlin/backend/0_scripts/server/build/libs/server-all.jar
2025-02-26 13:36:55.595 [main]  INFO  io.ktor.server.Application - Autoreload is disabled because the development mode is off.
2025-02-26 13:37:10.785 [main]  INFO  io.ktor.server.Application - Autoreload is disabled because the development mode is off.
2025-02-26 13:37:11.114 [main]  INFO  i.m.c.i.push.PushMeterRegistry - publishing metrics for LoggingMeterRegistry every 1m
2025-02-26 13:37:11.206 [main]  INFO  io.ktor.server.Application - Application started in 0.523 seconds.
Exception in thread "main" java.net.BindException: Address already in use
	at java.base/sun.nio.ch.Net.bind0(Native Method)
	at java.base/sun.nio.ch.Net.bind(Net.java:555)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.netBind(ServerSocketChannelImpl.java:337)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:294)
	at io.netty.channel.socket.nio.NioServerSocketChannel.doBind(NioServerSocketChannel.java:141)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.bind(AbstractChannel.java:561)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.bind(DefaultChannelPipeline.java:1281)
	at io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:600)
	at io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:579)
	at io.netty.channel.DefaultChannelPipeline.bind(DefaultChannelPipeline.java:922)
	at io.netty.channel.AbstractChannel.bind(AbstractChannel.java:259)
	at io.netty.bootstrap.AbstractBootstrap$2.run(AbstractBootstrap.java:380)
	at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:173)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:166)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:472)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:569)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:840)
2025-02-26 13:37:21.329 [main]  INFO  io.ktor.server.Application - Autoreload is disabled because the development mode is off.
2025-02-26 13:37:21.680 [main]  INFO  i.m.c.i.push.PushMeterRegistry - publishing metrics for LoggingMeterRegistry every 1m
2025-02-26 13:37:21.750 [main]  INFO  io.ktor.server.Application - Application started in 0.531 seconds.
Exception in thread "main" java.net.BindException: Address already in use
	at java.base/sun.nio.ch.Net.bind0(Native Method)
	at java.base/sun.nio.ch.Net.bind(Net.java:555)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.netBind(ServerSocketChannelImpl.java:337)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:294)
	at io.netty.channel.socket.nio.NioServerSocketChannel.doBind(NioServerSocketChannel.java:141)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.bind(AbstractChannel.java:561)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.bind(DefaultChannelPipeline.java:1281)
	at io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:600)
	at io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:579)
	at io.netty.channel.DefaultChannelPipeline.bind(DefaultChannelPipeline.java:922)
	at io.netty.channel.AbstractChannel.bind(AbstractChannel.java:259)
	at io.netty.bootstrap.AbstractBootstrap$2.run(AbstractBootstrap.java:380)
	at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:173)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:166)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:472)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:569)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:840)

2025-02-26 13:39:33.166 [main]  INFO  io.ktor.server.Application - Autoreload is disabled because the development mode is off.
2025-02-26T13:40:56.931971 | ERROR | Failed to check if port 80 is in use: Address already in use
2025-02-26T13:40:56.961661 | ERROR | Failed to check if port 80 is in use: Address already in use
2025-02-26T13:40:58.034981 | ERROR | Failed to check if port 80 is in use: Address already in use
2025-02-26 13:41:00.435 [main]  INFO  io.ktor.server.Application - Autoreload is disabled because the development mode is off.
2025-02-26T13:41:09.351966 | ERROR | Failed to check if port 80 is in use: Address already in use

2025-02-26T13:41:48.926163 | ERROR | Failed to check if port 80 is in use: Address already in use
2025-02-27 01:31:33.296 [main]  INFO  io.ktor.server.Application - Autoreload is disabled because the development mode is off.
2025-02-27T01:31:59.766107 | ERROR | Failed to check if port 80 is in use: Address already in use
2025-02-27T01:31:59.799688 | ERROR | Failed to check if port 80 is in use: Address already in use
2025-02-27T01:32:01.069911 | ERROR | Failed to check if port 80 is in use: Address already in use
2025-02-27 01:32:03.417 [main]  INFO  io.ktor.server.Application - Autoreload is disabled because the development mode is off.
2025-02-27T01:35:31.507713 | ERROR | Failed to check if port 80 is in use: Address already in use
2025-02-27T01:35:31.541993 | ERROR | Failed to check if port 80 is in use: Address already in use
2025-02-27T01:35:32.699661 | ERROR | Failed to check if port 80 is in use: Address already in use
2025-02-27 01:35:35.087 [main]  INFO  io.ktor.server.Application - Autoreload is disabled because the development mode is off.
2025-02-27T01:37:02.074426 | ERROR | Failed to check if port 80 is in use: Address already in use
2025-02-27 08:21:15.790 [main]  INFO  io.ktor.server.Application - Autoreload is disabled because the development mode is off.
2025-02-27T08:24:19.832607 | ERROR | Failed to check if port 80 is in use: Address already in use
2025-02-27T08:24:19.868844 | ERROR | Failed to check if port 80 is in use: Address already in use
2025-02-27T08:24:21.020055 | ERROR | Failed to check if port 80 is in use: Address already in use
nohup: .//Users/x/Desktop/kotlin/backend/server/build/install/server/bin/server: No such file or directory
nohup: ./Users/x/Desktop/kotlin/backend/server/build/install/server/bin/server: No such file or directory
nohup: ./Users/x/Desktop/kotlin/backend/0_scripts/../server/build/install/server/bin/server: No such file or directory
nohup: ./Users/x/Desktop/kotlin/backend/server/build/install/server/bin/server: No such file or directory
nohup: ./Users/x/Desktop/kotlin/backend/server/build/install/server/bin/server: No such file or directory
08:29:36,784 |-INFO in ch.qos.logback.classic.LoggerContext[default] - This is logback-classic version 1.5.11
08:29:36,785 |-INFO in ch.qos.logback.classic.util.ContextInitializer@bef2d72 - No custom configurators were discovered as a service.
08:29:36,785 |-INFO in ch.qos.logback.classic.util.ContextInitializer@bef2d72 - Trying to configure with ch.qos.logback.classic.joran.SerializedModelConfigurator
08:29:36,785 |-INFO in ch.qos.logback.classic.util.ContextInitializer@bef2d72 - Constructed configurator of type class ch.qos.logback.classic.joran.SerializedModelConfigurator
08:29:36,789 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback-test.scmo]
08:29:36,789 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback.scmo]
08:29:36,796 |-INFO in ch.qos.logback.classic.util.ContextInitializer@bef2d72 - ch.qos.logback.classic.joran.SerializedModelConfigurator.configure() call lasted 5 milliseconds. ExecutionStatus=INVOKE_NEXT_IF_ANY
08:29:36,796 |-INFO in ch.qos.logback.classic.util.ContextInitializer@bef2d72 - Trying to configure with ch.qos.logback.classic.util.DefaultJoranConfigurator
08:29:36,796 |-INFO in ch.qos.logback.classic.util.ContextInitializer@bef2d72 - Constructed configurator of type class ch.qos.logback.classic.util.DefaultJoranConfigurator
08:29:36,796 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback-test.xml]
08:29:36,798 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Found resource [logback.xml] at [jar:file:/Users/x/Desktop/kotlin/backend/server/build/install/server/lib/server-0.0.1.jar!/logback.xml]
08:29:36,799 |-WARN in ch.qos.logback.classic.util.DefaultJoranConfigurator@69b2283a - Resource [logback.xml] occurs multiple times on the classpath.
08:29:36,799 |-WARN in ch.qos.logback.classic.util.DefaultJoranConfigurator@69b2283a - Resource [logback.xml] occurs at [jar:file:/Users/x/Desktop/kotlin/backend/server/build/install/server/lib/server-0.0.1.jar!/logback.xml]
08:29:36,799 |-WARN in ch.qos.logback.classic.util.DefaultJoranConfigurator@69b2283a - Resource [logback.xml] occurs at [jar:file:/Users/x/Desktop/kotlin/backend/server/build/install/server/lib/swagger-codegen-generators-1.0.55.jar!/logback.xml]
08:29:36,802 |-INFO in ConfigurationWatchList(mainURL=jar:file:/Users/x/Desktop/kotlin/backend/server/build/install/server/lib/server-0.0.1.jar!/logback.xml, fileWatchList={}, urlWatchList=[}) - URL [jar:file:/Users/x/Desktop/kotlin/backend/server/build/install/server/lib/server-0.0.1.jar!/logback.xml] is not of type file
08:29:36,887 |-INFO in ch.qos.logback.core.model.processor.AppenderModelHandler - Processing appender named [STDOUT]
08:29:36,887 |-INFO in ch.qos.logback.core.model.processor.AppenderModelHandler - About to instantiate appender of type [ch.qos.logback.core.ConsoleAppender]
08:29:36,891 |-INFO in ch.qos.logback.core.model.processor.ImplicitModelHandler - Assuming default type [ch.qos.logback.classic.encoder.PatternLayoutEncoder] for [encoder] property
08:29:36,908 |-INFO in ch.qos.logback.classic.model.processor.RootLoggerModelHandler - Setting level of ROOT logger to INFO
08:29:36,908 |-INFO in ch.qos.logback.core.model.processor.AppenderRefModelHandler - Attaching appender named [STDOUT] to Logger[ROOT]
08:29:36,908 |-INFO in ch.qos.logback.classic.model.processor.LoggerModelHandler - Setting level of logger [org.eclipse.jetty] to INFO
08:29:36,908 |-INFO in ch.qos.logback.classic.model.processor.LoggerModelHandler - Setting level of logger [io.netty] to INFO
08:29:36,908 |-INFO in ch.qos.logback.core.model.processor.DefaultProcessor@1d119efb - End of configuration.
08:29:36,909 |-INFO in ch.qos.logback.classic.joran.JoranConfigurator@659a969b - Registering current configuration as safe fallback point
08:29:36,909 |-INFO in ch.qos.logback.classic.util.ContextInitializer@bef2d72 - ch.qos.logback.classic.util.DefaultJoranConfigurator.configure() call lasted 113 milliseconds. ExecutionStatus=DO_NOT_INVOKE_NEXT_IF_ANY
2025-02-27T08:30:00.412248 | ERROR | Failed to check if port 80 is in use: Address already in use
2025-02-27T08:31:25.764766 | ERROR | Failed to check if port 80 is in use: Address already in use
